enriched.c

```diff
@@ -282,7 +282,7 @@ static void enriched_putwc(wchar_t c, struct EnrichedState *enriched)
     if (enriched->tag_level[RICH_COLOR])
     {
       if ((enriched->param_used + 1) >= enriched->param_len)
-        mutt_mem_realloc(&enriched->param, (enriched->param_len += 256) * sizeof(wchar_t));
+        MUTT_MEM_REALLOC(&enriched->param, (enriched->param_len += 256), wchar_t);
 
       enriched->param[enriched->param_used++] = c;
     }
```

Comment:
@flatcap Do you prefer if I append a commit splitting the `+=` into a separate line?

Reply:
Yes, please.

Reply:
Done.